llvm/lib/CodeGen/MachineTraceMetrics.cpp

```diff
@@ -133,7 +127,7 @@ MachineTraceMetrics::getResources(const MachineBasicBlock *MBB) {
 
   // Scale the resource cycles so they are comparable.
   unsigned PROffset = MBB->getNumber() * PRKinds;
-  for (unsigned K = 0; K != PRKinds; ++K)
+  for (unsigned K = 0; K < PRKinds; ++K)
```

Comment:
The coding standard doesn't specify which style of control condition is preferred, so there is no reason to change this line.